backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -50,15 +47,13 @@ async function calculateAndAddDecayTransactions(
     transactionIds.push(userTransaction.transactionId)
   })
 
-  const transactionRepository = getCustomRepository(TransactionRepository)
-  const transactions = await transactionRepository.joinFullTransactionsByIds(transactionIds)
-
+  const transactions = await dbTransaction.find({ where: { id: In(transactionIds) } })
   const transactionIndiced: dbTransaction[] = []
   transactions.forEach((transaction: dbTransaction) => {
     transactionIndiced[transaction.id] = transaction
+    involvedUserIds.push(transaction.userId)
     if (transaction.transactionTypeId === TransactionTypeId.SEND) {
-      involvedUserIds.push(transaction.transactionSendCoin.userId)
-      involvedUserIds.push(transaction.transactionSendCoin.recipiantUserId)
+      involvedUserIds.push(transaction.sendReceiverUserId!) // TODO ensure not null properly
```

Comment:
What is the exclamation mark at the end?

Reply:
This is an assertion not to be null. It should throw in other cases. This isnot very nice, but the combination of the tables allows null on the field, since we also have creation transaction in the same table.
I would like to keep it till we have a better solution how to store transactions. Hence the Todo